lib/matplotlib/tests/test_backend_pgf.py

```diff
@@ -78,7 +78,7 @@ def create_figure():
 
 
 # test compiling a figure to pdf with xelatex
-@cleanup(style='classic')
+@cleanup(style='_classic_test')
```

Comment:
The tests in this file don't actually use `classic`, as noted in the pytest conversion; maybe just set them to `default` now to reflect what's really going on?